.github/workflows/delete_nais.yaml

```diff
@@ -23,17 +23,17 @@ jobs:
           echo "ISSUE_NUMBER=${ISSUE_NUMBER}" >> $GITHUB_ENV;
           echo "Issue number: $ISSUE_NUMBER";
 
-      - id: "auth"
-        name: "Authenticate to Google Cloud"
-        uses: "google-github-actions/auth@v0.6.0"
+      - id: 'auth'
+        name: 'Authenticate to Google Cloud'
+        uses: 'google-github-actions/auth@v2.1.6'
         with:
-          workload_identity_provider: "projects/906675412832/locations/global/workloadIdentityPools/ssb-identity-pool/providers/github-oidc-provider"
-          service_account: "gh-ssb@nais-management-b3a7.iam.gserviceaccount.com"
-          token_format: "access_token"
-      - id: "get-credentials"
+          workload_identity_provider: 'projects/906675412832/locations/global/workloadIdentityPools/ssb-identity-pool/providers/github-oidc-provider'
```

Comment:
We need to look into whether `workload_indentity_provider` and `service_account` should be secured as variables since they can contain sensitive information

Reply:
There is a task in the Backlog that adresses the issue